protostar/starknet_gateway/block_explorer/stark_scan_block_explorer.py

```diff
@@ -0,0 +1,28 @@
+from protostar.starknet_gateway.network_config import PredefinedNetwork
+
+from .block_explorer import (
+    URL,
+    BlockExplorer,
+    ClassHash,
+    ContractAddress,
+    TransactionHash,
+)
+
+
+class StarkScanBlockExplorer(BlockExplorer):
+    def __init__(self, network: PredefinedNetwork) -> None:
+        super().__init__()
+        network_to_domain: dict[PredefinedNetwork, URL] = {
+            "mainnet": "https://starkscan.co",
+            "testnet": "https://testnet.starkscan.co",
```

Comment:
What about testnet2?

Reply:
It's out of scope of this PR. Starknet.py needs to introduce it.

Reply:
Starknet.py team says they will rather not introduce anything to PredefinedNetwork. But I think this doesn't mean we should also not?

Reply:
Then we need to decouple this concept from Starknet.py. That's a different issue.